deps.edn

```diff
@@ -11,7 +11,7 @@
          com.taoensso/timbre         {:mvn/version "4.10.0"}
          hickory                     {:mvn/version "0.7.1"}
          com.cognitect/transit-cljs  {:mvn/version "0.8.248"}
-         status-im/pluto             {:mvn/version "iteration-4-3"}
+         status-im/pluto             {:mvn/version "iteration-4-5"}
```

Comment:
Why not 4.4?

Reply:
still confusing with the version, can change to 4.4